pkgs/by-name/on/ondsel/package.nix

```diff
@@ -0,0 +1,51 @@
+{ appimageTools, lib, stdenv, fetchurl }:
+
+let
+  pname = "ondsel";
+  version = "2024.1.0";
+  patch = "35694";
+
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  suffix = {
+    x86_64-linux = "Linux-x86_64.AppImage";
+    # aarch64-linux = "Linux-aarch64.AppImage";
+    # x86_64-darwin = "macOS-intel-x86_64.dmg";
+    # aarch64-darwin = "macOS-apple-silicon-arm64.dmg";
+  }.${system} or throwSystem;
+
+  src = fetchurl {
+    # TODO: integrate version rest...
```

Comment:
This todo message isn't very clear. Maybe you could elaborate?

Reply:
The current build has a `patch modifier` (maybe some kind of timestamp?) in the url besides the official `major-minor-patch` version (2024.1.0) that I think is going to change in the future. I'm not quite sure if I should integrate it into the url or keep it as a separate variable as it is now. Therefore this TODO